consumer/pdata/metric.go

```diff
@@ -51,6 +52,13 @@ func MetricsToOtlp(md Metrics) []*otlpmetrics.ResourceMetrics {
 	return *md.orig
 }
 
+// MetricsToOtlpProtoBytes converts the internal MetricData to the OTLP Collector ExportMetricsServiceRequest ProtoBuf bytes
+func MetricsToOtlpProtoBytes(md Metrics) ([]byte, error) {
```

Comment:
Why not just a func in the Metrics struct?

Reply:
Same applies to all structs

Reply:
Sure, we can change all of them into methods instead of function.

I'll change to use `ToOtlpProtoBytes()` methods on the three structs.